stratum/hal/lib/tdi/tdi_sde_interface.h

```diff
@@ -126,9 +129,18 @@ class TdiSdeInterface {
     // Sets a table data action parameter.
     virtual ::util::Status SetParam(int id, const std::string& value) = 0;
 
+    // Sets a table data action parameter.
+    virtual ::util::Status SetParam(std::string field_name, uint64 value) = 0;
+
+    // Sets a table data action parameter.
+    virtual ::util::Status SetParam(std::string field_name, const std::string& value) = 0;
```

Comment:
`const std::string &field_name`

Reply:
Yes, I believe this can be changed without any impact